app/controllers/external_api_application_controller.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+# controllers that aren't a part of the vets-website backend should inherit from this controller
+class ExternalApiApplicationController < ApplicationController
+  skip_before_action :verify_authenticity_token
+  skip_after_action :set_csrf_header
+end
```

Comment:
I'm not fond of this pattern, that is, adding inheritance between a controller and application_controller. It quickly gets unreadable once there's more than a couple of layers of inheritance up until application_controller.

Have you considered using a mixin instead? Something like, `include CsrfProtection`, and then have:

```ruby
included do
  skip_before_action :verify_authenticity_token
  skip_after_action :set_csrf_header
end
```

Reply:
Also, I'm not sure how realistic it is to expect that other developers will know that this controller exists (i.e. `controllers that aren't a part of the vets-website backend should inherit from this controller`). I'd imagine they would just manually write

```ruby
  skip_before_action :verify_authenticity_token
  skip_after_action :set_csrf_header
```
at the top of their controllers.

Reply:
Refactored and removed `ExternalApiApplicationController` controller. 
Whether it's using a new parent controller or having to manually `skip` the before filters, there's too much implicit universal (for Lighthouse) required code here that should be built in to all Lighthouse endpoints.